cluster.go

```diff
@@ -1817,6 +1835,28 @@ func (c *cluster) nodeLeave(nodeID string) error {
 	return nil
 }
 
+func (c *cluster) nodeStatus() *NodeStatus {
+	ns := &NodeStatus{
+		Node:   c.Node,
+		Schema: &Schema{Indexes: c.holder.Schema()},
+	}
+	for _, idx := range ns.Schema.Indexes {
+		is := &IndexStatus{Name: idx.Name}
+		for _, f := range idx.Fields {
+			availableShards := roaring.NewBitmap()
```

Comment:
I think it would be better to move declaration of `availableShards` to above the topmost loop like `var availableShards roaring.Bitmap` and add an `else` clause to the `if` below which sets it to `availableShards = roaring.Bitmap` in order to get rid of a potentially not used allocation.